src/test/java/BaseTest.java

```diff
@@ -96,4 +101,32 @@ public void addSongNewList(String playlistName) {
         enterElement.click();
     }
 
+    public void editPlaylistName() {
+
+        WebElement findExisting = driver.findElement(By.xpath(".//a[contains(text(),\"CC's new playlist\")]"));
```

Comment:
Please finish the homework!You got this problem, please correct it. 


![image](https://user-images.githubusercontent.com/104761482/236145471-e8839c30-210b-4f88-997b-7814629ae890.png)




HINT

Hence, press Control+F. A text field will appear. Write the expression and check if it matches with an element in the webpage. If it is matching, then copy the expression and paste in selenium code so we can avoid NoSuchElemetException

![image](https://user-images.githubusercontent.com/104761482/236145392-6144ae36-1223-4332-81b5-22ff4128b791.png)